### DISCUSSION TITLE
Debugging with SubscriptionID

### DISCUSSION THREAD

**Discussion by SamLyuubc** [FreeOpcUa/opcua-asyncio#770] [Q&A]
I found a issue on our application where a subscription's publish response did not received a publish request with acknowledgement. By using wireshark, I am able to find the subscriptionID. How can I further identify which node's subscription is not receiving acknowledgement. Ideally, I would like to know the node name and namespace. Thanks for your help and suggestions.

In the attached pictures, you can see the subscriptionid 129 has a request with acknowledgement however subscriptionID 128 did not receive any acknowledgement. Thanks.

![response_129](https://user-images.githubusercontent.com/23411719/149385600-97833334-b6a1-45f1-87d2-227fda03f992.png)
![request_129](https://user-images.githubusercontent.com/23411719/149385608-4ec97d0f-c3df-43b8-b09f-3e041567e2a8.png)
![response_128](https://user-images.githubusercontent.com/23411719/149385579-e16077f2-884d-422d-8372-9aa1bdf87579.png)
![request_128](https://user-images.githubusercontent.com/23411719/149385584-6a50b4ef-61bd-406d-91fc-1abd5690eb62.png)

**Comment by oroulet** [comment]
what client and what server? ack is implemented server side so if this is not an asyncua server we cannot do much.

**Reply by SamLyuubc** [reply]
Hey it's a asyncua server with a ignition client.
